.travis.yml

```diff
@@ -1,13 +1,23 @@
 sudo: false
 language: scala
+services:
+- redis-server
```

Comment:
Our travis build is already configured to talk to an existing Redis server. What is this for?